File: Rokovi/app/src/main/java/tzilic_20/rokovi/model/Rok.java
package tzilic_20.rokovi.model;

import java.io.Serializable;
import java.util.Date;

public class Rok implements Serializable {

    private int sifra;
    private Date datum;
    private String kolegij;
    private int brojPrijavljenih;

    public Rok() {
    }

    public Rok(int sifra, Date datum, String kolegij, int brojPrijavljenih) {
        this.sifra = sifra;
        this.datum = datum;
        this.kolegij = kolegij;
        this.brojPrijavljenih = brojPrijavljenih;
    }

    public int getSifra() {
        return sifra;
    }

    public void setSifra(int sifra) {
        this.sifra = sifra;
    }

    public Date getDatum() {
        return datum;
    }

    public void setDatum(Date datum) {
        this.datum = datum;
    }

    public String getKolegij() {
        return kolegij;
    }

    public void setKolegij(String kolegij) {
        this.kolegij = kolegij;
    }

    public int getBrojPrijavljenih() {
        return brojPrijavljenih;
    }

    public void setBrojPrijavljenih(int brojPrijavljenih) {
        this.brojPrijavljenih = brojPrijavljenih;
    }
}

File: RESTAPI/src/tzilic_20/Rok.php
<?php
namespace tzilic_20;

use DateTime;

/**
 * @Entity @Table(name="rokovi")
 **/
class Rok
{
    /** @Id @Column(type="integer") @GeneratedValue  **/
    private $sifra;

    /** @Column(type="date") **/
    private $datum;

    /** @Column(type="string") **/
	   private $kolegij;

     /** @Column(type="integer") **/
    private $brojPrijavljenih;

	public function __construct($podaci=null)
	{
		if($podaci==null){
			return;
		}
    $date = new DateTime($podaci->datum);
    $this->datum = $date;
		$this->kolegij=$podaci->kolegij;
    $this->brojPrijavljenih=$podaci->brojPrijavljenih;
	}

  public function getSifra(){
  return $this->sifra;
}

public function setSifra($sifra){
  $this->sifra = $sifra;
}

public function getDatum(){
  return $this->datum;
}

public function setDatum($datum){
  $date = new \DateTime($datum);
  $this->datum = $date;
}

public function getKolegij(){
  return $this->kolegij;
}

public function setKolegij($kolegij){
  $this->kolegij = $kolegij;
}

public function getBrojPrijavljenih(){
  return $this->brojPrijavljenih;
}

public function setBrojPrijavljenih($brojPrijavljenih){
  $this->brojPrijavljenih = $brojPrijavljenih;
}

}
